ui_framework/src/components/button/_button.scss

```diff
@@ -0,0 +1,123 @@
+// Our base button
+.kuiButton {
+  display: inline-block;
+  appearance: none;
+  cursor: pointer;
+  height: $kuiSizeXXL;
+  text-decoration: none;
+  border: solid 1px $kuiColorPrimary;
+  border-radius: $kuiBorderRadius;
+  color: $kuiColorPrimary;
+  padding: 0 $kuiSizeS;
+  min-width: $kuiSize * 7;
+  text-align: center;
+  font-family: $kuiFontFamily;
+  transition: all $kuiAnimSpeedNormal $kuiAnimSlightBounce;
+  whitespace: nowrap;
+
+  &.kuiButton--small {
+    height: $kuiSizeXL;
+  }
+
+  .kuiButton__icon {
+    vertical-align: -2px;
+    fill: $kuiColorPrimary;
+  }
+
+  &:hover, &:focus {
+    text-decoration: underline;
+    background-color: transparentize($kuiColorPrimary, .9);
+  }
+
+  &:focus {
```

Comment:
![](https://d3vv6lp55qjaqc.cloudfront.net/items/1B410T2c3o0b0E3b2V2P/Screen%20Recording%202017-08-01%20at%2008.13%20PM.gif?X-CloudApp-Visitor-Id=59773&v=491733d3) Solved this by using an animation instead. Now it will pop back. Might still add an outline to it as well, but should be decent for now.